tests/lib/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+CACHE_DIR="$TESTSLIB/snaps/cache"
```

Comment:
Can we please turn this into a simpler library routine in `snaps.sh`:
```
#!/bin/sh

install_local() {
    local SNAP_FILE="$TETSLIB/snaps/$1_1.0_all.snap"
    if [ ! -f "$SNAP_FILE" ]; then
            snapbuild $(dirname $SNAP_FILE)
    fi
    snap install --dangerous "$SNAP_FILE"
}
```
This way we can use `. $TESTLIB/snaps.sh`, and then use `install_local snap_name` as a command.